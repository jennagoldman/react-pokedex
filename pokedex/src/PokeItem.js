import React, { Component } from 'react';

export default class PokeItem extends Component {
    render() {
        return (
                <li className="pokemon-item">
                    <h2>{this.props.pokemon.pokemon.toUpperCase()}</h2>
                    <img src={this.props.pokemon.url_image} alt={this.props.pokemon.pokemon} />
                    <p>HP: {this.props.pokemon.hp}</p>
                    <p>Types: 
                        <span> {this.props.pokemon.type_1}</span>
                        <span>, {this.props.pokemon.type_2} </span>
                    </p>
                </li>
        )
    }
}