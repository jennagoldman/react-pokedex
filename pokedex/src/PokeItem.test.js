import React from 'react';
import PokeItem from './PokeItem';
import pokemonData from './data'
import renderer from 'react-test-renderer';

it('App renders correctly', () => {
  const tree = renderer
    .create(<PokeItem pokemon={pokemonData} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});