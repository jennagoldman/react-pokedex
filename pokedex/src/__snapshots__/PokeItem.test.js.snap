// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<li
  className="pokemon-item"
>
  <h2 />
  <img />
  <p>
    HP: 
  </p>
  <p>
    Types:
    <span>
       
    </span>
    <span>
      , 
       
    </span>
  </p>
</li>
`;
